pkg/storage/unified/sql/data/resource_update.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE {{ .Ident "resource" }}
+    SET
+        {{ .Ident "guid" }}  = {{ .Arg .GUID }},
+        {{ .Ident "value" }}  = {{ .Arg .WriteEvent.Value }},
+        {{ .Ident "action" }} = {{ .Arg .WriteEvent.Type }}  
+    WHERE 1 = 1
+        AND   {{ .Ident "group" }}    = {{ .Arg .WriteEvent.Key.Group }}
+        AND   {{ .Ident "resource" }}  = {{ .Arg .WriteEvent.Key.Resource }}
+        AND   {{ .Ident "namespace" }} = {{ .Arg .WriteEvent.Key.Namespace }}
+        AND   {{ .Ident "name" }}      = {{ .Arg .WriteEvent.Key.Name }} 
```

Comment:
Can we do something about the inconsistent whitespace in these templates? Here we have both misaligned = where we're trying to put them in a column and extra spaces after AND. Both are presumably copypaste error but this kind of thing creates messy diffs when someone fixes them manually.